homeassistant/components/avea/light.py

```diff
@@ -0,0 +1,96 @@
+"""Support for the Elgato Avea lights."""
+import logging
+
+from homeassistant.components.light import (
+    ATTR_BRIGHTNESS, ATTR_HS_COLOR, SUPPORT_BRIGHTNESS,
+    SUPPORT_COLOR, Light)
+from homeassistant.exceptions import PlatformNotReady
+import homeassistant.util.color as color_util
+
+
+_LOGGER = logging.getLogger(__name__)
+
+SUPPORT_AVEA = (SUPPORT_BRIGHTNESS | SUPPORT_COLOR)
+
+
+def setup_platform(hass, config, add_devices, discovery_info=None):
```

Comment:
Please rename `add_devices` to `add_entities`.

Reply:
Thanks, fixed in 781d19821460edf8c6a089c9d8f8858ac0cf9462 👍